test/access-token.test.js

```diff
@@ -280,7 +280,7 @@ describe('loopback.token(options)', function() {
       ' when enableDoubkecheck is true',
     function(done) {
       var token = this.token;
-
+      var app = loopback({localRegistry: true, loadBuiltinModels: true});
```

Comment:
This looks rather confusing to me, as this line is creating a local model registry but the test itself uses token in the global registry.

The correct solution is to rework the infrastructure in this test file to use a local per-app registry for everything and drop global `AccessToken` (see lines 18-21).

As a short-term fix to unblock your eslint work, I am proposing to change this line to `var app = loopback();` and if that does not work, then `var app = express();`.